source/common/vue/form/elements/Checkbox.vue

```diff
@@ -1,16 +1,26 @@
 <template>
-  <div class="form-control cb-group">
-    <label class="checkbox">
-      <input
-        v-bind:id="fieldID"
-        type="checkbox" v-bind:name="name" value="yes"
-        v-bind:checked="value"
-        v-bind:disabled="disabled"
-        v-on:input="$emit('input', $event.target.checked)"
+  <div>
+    <div class="form-control cb-group">
+      <label class="checkbox"
+             v-bind:disabled="disabled"
       >
-      <span class="checkmark"></span>
-    </label>
-    <label v-if="label" v-bind:for="fieldID" v-html="label"></label>
+        <input
+          v-bind:id="fieldID"
+          type="checkbox" v-bind:name="name" value="yes"
+          v-bind:checked="value"
+          v-bind:disabled="disabled"
+          v-on:input="$emit('input', $event.target.checked)"
+        >
+        <span class="checkmark"></span>
+      </label>
+      <label v-if="label" v-bind:for="fieldID" v-bind:disabled="disabled"
+             v-html="label"
+      >
+      </label>
+    </div>
+    <div v-if="additionaltext" class="form-control">
+      {{ additionaltext }}
```

Comment:
Instead of "additionaltext" I would suggest you call this either "infoString" or just "info" – this makes more sense and explains what this precisely does. Plus I remember some elements support (or have supported?) this already and I'm pretty sure I named it this way.